lib/round.rb

```diff
@@ -0,0 +1,96 @@
+require './lib/deck'
+
+class Round
+  attr_reader :deck,
+              :turns
+  attr_accessor :correct_answers,
+                :turn_count,
+                :total_cards,
+                :categories
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @correct_answers = 0
+    @turn_count = 1
+    @total_cards = 0
+    @categories = []
+  end
+
+  def current_card
+    @deck.cards.first
+  end
+
+  def take_turn(answer, card = current_card)
+    @deck.cards.shift
+    @turn_count += 1
+    new_turn = Turn.new(answer, card)
+    @turns << new_turn
+    @correct_answers += 1 if @turns.last.correct?
+    new_turn
+  end
+
+  def number_correct
+    @correct_answers
+  end
```

Comment:
you could change the instance variable to be called `@number_correct`, and then keep an attr_reader for it so you can read it from other classes.

Reply:
^^ if you did that, you wouldn't need this method